source/installation/kafka-deployments/migrator-with-kafka.txt

```diff
@@ -0,0 +1,81 @@
+.. _kafka-intro:
+
+======================================
+Relational Migrator with Kafka Connect
+======================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use Relational Migrator to migrate data to and from Kafka. 
```

Comment:
This sentence is inaccurate. Relational Migrator is still migrating data from a source relational database and writing to MongoDB. Kafka is basically being used as a robust transport layer between the components.

Also can we refer to the product as "Apache Kafka" on first mention?

Reply:
Updated.